service/src/main/kotlin/fi/espoo/evaka/emailclient/EmailClient.kt

```diff
@@ -34,7 +34,10 @@ private constructor(
             traceId: String,
         ): Email? {
             val (toAddress, enabledEmailTypes) =
-                dbc.read { tx -> tx.getEmailAddressAndEnabledTypes(personId) }
+                dbc.read { tx ->
+                    tx.getEmailAddressAndEnabledTypes(personId)
+                        ?: EmailAndEnabledEmailTypes(null, null)
```

Comment:
optional nitpick: this should rather log that the person was not found (which sounds unexpected and a possible bug) and return